src/pages/Policy/index.tsx

```diff
@@ -1,12 +1,25 @@
-import TextBox from "@components/TextBox";
 import { Sc } from "./style";
-
+import { Gs } from "src/styles/globalStyles";
+import Terms from "./Terms";
+import Entypo from "react-native-vector-icons/Entypo";
+import { NavigationType } from "@type/routeType";
+import { useNavigation } from "@react-navigation/native";
 const Policy: React.FC = () => {
+  const navigation = useNavigation<NavigationType>();
   return (
     <Sc.Container nestedScrollEnabled contentContainerStyle={{ flexGrow: 1 }}>
+      <Entypo
+        style={{ position: "absolute", top: 0, left: -10 }}
+        name="chevron-left"
+        size={40}
+        color="#9BABB7"
+        onPress={() => navigation.navigate("SignUp")}
+      />
       <Sc.Wrapper>
-        <TextBox>cia-65: Tela de termos de uso ou na Política de Privacidade</TextBox>
+        <Gs.Title>{`Termos de Uso e \nPolítica de Privacidade`}</Gs.Title>
```

Comment:
Aqui é interessante não forçar uma quebra de linha. Provavelmente fez pra ficar igual o figma, mas se um usuário tiver a fonte aumentada (muito comum), isso vai acabar bugando. Eu aumentei a do meu celular pra testar, e ficou assim: 
```
< Termos de 
Uso e 
Politíca...
```

e sem a quebra ele fica 
```
< Termos de Uso e 
Política ...
```